Management

```diff
@@ -0,0 +1,37 @@
+gantt
+title STAFF ALLOCATION CHART
+    dateFormat  YYYY-MM-DD
+    section TEAM
+    Group Setting     :done, a1, 2020-10-16, 10d
+    Topic Choice  : done, a2, after a1  , 5d
+    Project Overview  : done, a3, 2020-10-30, 1d
+    Staff Upskill  :active, a17, after a3, 48d
+    Report production :a4, after a12  , 14d
+    Report release    :a5, 2020-12-18  , 1d
+    Design  :a20, after a5, 48d
```

Comment:
If we say the Design starts earlier (e.g. Nov 2020), while the UML is being developed, in the report we can say that we work in an agile manner where implementation demos are used to feedback into the requirements development

Reply:
gantt
title ECG Smartwatch Design
    dateFormat  YYYY-MM-DD
    section ASSIGNMENT
    Group Setting     :done, a1, 2020-10-16, 10d
    Topic Choice  : done, a2, after a1  , 5d
    Project Overview  : done, a3, 2020-10-30, 1d
    Report production :a4, after a12  , 16d
    Report release    :a5, 2020-12-18  , 1d
    Design  :a20, after a6, 40d
    section SYSTEM
    System Architecture  :active, a6, 2020-11-02  , 7d
    User Characteristics :active, a7, 2020-11-09  , 6d
    Assumptions          :active, a8, 2020-11-09  , 3d
    section REQUIREMENTS
    Functional        :active, a9, 2020-11-09, 27d
    Non-Functional    :active, a10, 2020-11-15, 21d
    section DIAGRAMS
    Overall           :active, a12, 2020-11-09, 25d
    Use Case          :active, a13, 2020-11-13, 10d
    State             :a20, after a13, 10d
    Class             :a14, after a13, 10d
    Interaction       :a15, after a13, 10d
    Activity          :a16, 2020-11-30, 8d
    section MANAGEMENT
    Staff Upskill  :active, a17, after a3, 48d
    Management Chart  :done, a18, after a3, 20d 
    Risk assessment   :active, a19, after a6, 30d
    section IMPLEMENTATION @jdrew1303
    section INTEGRATION @jdrew1303
    Validation  :a21, after a20, 48d
Product Launch  :a22, after a21, 5d
    section OPERATION AND MAINTENANCE
    Support  :a17, after a22, 5d

Reply:
Ok for me, I propose to add that for Saturday, and let @jdrew1303 James update it later in a new branch.

Reply:
Accepted
![mermaid-diagram-20201209222719](https://user-images.githubusercontent.com/60558454/101696395-f6b78280-3a6d-11eb-8793-26fbe68d6293.png)

```suggestion 
    Design  :a20, after a5, 48d
```

Reply:
Hey I've added my changes based on Jeramy's work above:

[![](https://mermaid.ink/img/eyJjb2RlIjoiZ2FudHRcbnRpdGxlIEVDRyBTbWFydHdhdGNoXG5kYXRlRm9ybWF0ICBZWVlZLU1NLUREXG5cbnNlY3Rpb24gQVNTSUdOTUVOVFxuICAgIEdyb3VwIFNldHRpbmcgICAgIDpkb25lLCBhMSwgMjAyMC0xMC0xNiwgMTBkXG4gICAgVG9waWMgQ2hvaWNlICA6IGRvbmUsIGEyLCBhZnRlciBhMSAgLCA1ZFxuICAgIFByb2plY3QgT3ZlcnZpZXcgIDogZG9uZSwgYTMsIDIwMjAtMTAtMzAsIDFkXG4gICAgUmVwb3J0IHByb2R1Y3Rpb24gOmE0LCBhZnRlciBhMTIgICwgMTZkXG4gICAgUmVwb3J0IHJlbGVhc2UgICAgOmE1LCAyMDIwLTEyLTE4ICAsIDFkXG5cbnNlY3Rpb24gU1lTVEVNXG4gICAgU3lzdGVtIEFyY2hpdGVjdHVyZSAgOmFjdGl2ZSwgYTYsIDIwMjAtMTEtMDIgICwgN2RcbiAgICBVc2VyIENoYXJhY3RlcmlzdGljcyA6YWN0aXZlLCBhNywgMjAyMC0xMS0wOSAgLCA2ZFxuICAgIEFzc3VtcHRpb25zICAgICAgICAgIDphY3RpdmUsIGE4LCAyMDIwLTExLTA5ICAsIDNkXG5cbnNlY3Rpb24gUkVRVUlSRU1FTlRTXG4gICAgRnVuY3Rpb25hbCAgICAgICAgOmFjdGl2ZSwgYTksIDIwMjAtMTEtMDksIDI3ZFxuICAgIE5vbi1GdW5jdGlvbmFsICAgIDphY3RpdmUsIGExMCwgMjAyMC0xMS0xNSwgMjFkXG5cbnNlY3Rpb24gRElBR1JBTVNcbiAgICBPdmVyYWxsICAgICAgICAgICA6YWN0aXZlLCBhMTIsIDIwMjAtMTEtMDksIDI1ZFxuICAgIFVzZSBDYXNlICAgICAgICAgIDphY3RpdmUsIGExMywgMjAyMC0xMS0xMywgMTBkXG4gICAgU3RhdGUgICAgICAgICAgICAgOmEyMCwgYWZ0ZXIgYTEzLCAxMGRcbiAgICBDbGFzcyAgICAgICAgICAgICA6YTE0LCBhZnRlciBhMTMsIDEwZFxuICAgIEludGVyYWN0aW9uICAgICAgIDphMTUsIGFmdGVyIGExMywgMTBkXG4gICAgQWN0aXZpdHkgICAgICAgICAgOmExNiwgMjAyMC0xMS0zMCwgOGRcblxuc2VjdGlvbiBNQU5BR0VNRU5UXG4gICAgU3RhZmYgVXBza2lsbCAgOmFjdGl2ZSwgYTE3LCBhZnRlciBhMywgNDhkXG4gICAgTWFuYWdlbWVudCBDaGFydCAgOmRvbmUsIGExOCwgYWZ0ZXIgYTMsIDIwZCBcbiAgICBSaXNrIGFzc2Vzc21lbnQgICA6YWN0aXZlLCBhMTksIGFmdGVyIGE2LCAzMGRcblxuc2VjdGlvbiBJTVBMRU1FTlRBVElPTlxuICAgIENvcmUgQXJjaGl0ZWN0dXJlICA6YTMwLCBhZnRlciBhNCwgMndcbiAgICBSb3V0aW5nICYgTmF2aWdhdGlvbiAgOmEzMSwgYWZ0ZXIgYTMwLCAxd1xuICAgIENsb2NrICA6YTMyLCBhZnRlciBhMzEsIDNkXG4gICAgRGlzcGxheSAgOmEzMywgYWZ0ZXIgYTMyLCAxd1xuICAgIEJsdWV0b290aCAgOmEzNCwgYWZ0ZXIgYTMzLCAxd1xuICAgIFNldHRpbmdzIFNjcmVlbiAgOmEzNSwgYWZ0ZXIgYTM0LCAzZFxuICAgIFBvd2VyIExldmVscyAgOmEzNiwgYWZ0ZXIgYTM0LCAxd1xuICAgIEVDRyA6YTM3LCBhZnRlciBhMzQsIDF3XG4gICAgSGVhcnQgUmF0ZSA6YTM4LCBhZnRlciBhMzQsIDF3XG4gICAgTWFwcyA6YTM5LCBhZnRlciBhMzQsIDhkXG4gICAgUGVkb21ldGVyIDphNDAsIGFmdGVyIGEzNCwgMXdcbiAgICBTcE8yIDphNDEsIGFmdGVyIGEzNCwgMXdcblxuc2VjdGlvbiBJTlRFR1JBVElPTlxuICAgIFVzZXIgQWNjZXB0YW5jZSBUZXN0aW5nICA6YTQyLCBhZnRlciBhNDEsIDR3XG4gICAgU2VjdXJpdHkgVGVzdGluZyAgOmE0MywgYWZ0ZXIgYTQyLCAyd1xuICAgIERvY3VtZW50YXRpb24gIDphNDQsIGFmdGVyIGE0MSwgNXdcblxuc2VjdGlvbiBPUEVSQVRJT04gQU5EIE1BSU5URU5BTkNFXG4gICAgUHJvZHVjdCBMYXVuY2ggIDphMjIsIGFmdGVyIGE0MywgNWRcbiAgICBTdXBwb3J0ICA6YTE3LCBhZnRlciBhMjIsIDVkIiwibWVybWFpZCI6eyJ0aGVtZSI6ImRlZmF1bHQifSwidXBkYXRlRWRpdG9yIjpmYWxzZX0)](https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZ2FudHRcbnRpdGxlIEVDRyBTbWFydHdhdGNoXG5kYXRlRm9ybWF0ICBZWVlZLU1NLUREXG5cbnNlY3Rpb24gQVNTSUdOTUVOVFxuICAgIEdyb3VwIFNldHRpbmcgICAgIDpkb25lLCBhMSwgMjAyMC0xMC0xNiwgMTBkXG4gICAgVG9waWMgQ2hvaWNlICA6IGRvbmUsIGEyLCBhZnRlciBhMSAgLCA1ZFxuICAgIFByb2plY3QgT3ZlcnZpZXcgIDogZG9uZSwgYTMsIDIwMjAtMTAtMzAsIDFkXG4gICAgUmVwb3J0IHByb2R1Y3Rpb24gOmE0LCBhZnRlciBhMTIgICwgMTZkXG4gICAgUmVwb3J0IHJlbGVhc2UgICAgOmE1LCAyMDIwLTEyLTE4ICAsIDFkXG5cbnNlY3Rpb24gU1lTVEVNXG4gICAgU3lzdGVtIEFyY2hpdGVjdHVyZSAgOmFjdGl2ZSwgYTYsIDIwMjAtMTEtMDIgICwgN2RcbiAgICBVc2VyIENoYXJhY3RlcmlzdGljcyA6YWN0aXZlLCBhNywgMjAyMC0xMS0wOSAgLCA2ZFxuICAgIEFzc3VtcHRpb25zICAgICAgICAgIDphY3RpdmUsIGE4LCAyMDIwLTExLTA5ICAsIDNkXG5cbnNlY3Rpb24gUkVRVUlSRU1FTlRTXG4gICAgRnVuY3Rpb25hbCAgICAgICAgOmFjdGl2ZSwgYTksIDIwMjAtMTEtMDksIDI3ZFxuICAgIE5vbi1GdW5jdGlvbmFsICAgIDphY3RpdmUsIGExMCwgMjAyMC0xMS0xNSwgMjFkXG5cbnNlY3Rpb24gRElBR1JBTVNcbiAgICBPdmVyYWxsICAgICAgICAgICA6YWN0aXZlLCBhMTIsIDIwMjAtMTEtMDksIDI1ZFxuICAgIFVzZSBDYXNlICAgICAgICAgIDphY3RpdmUsIGExMywgMjAyMC0xMS0xMywgMTBkXG4gICAgU3RhdGUgICAgICAgICAgICAgOmEyMCwgYWZ0ZXIgYTEzLCAxMGRcbiAgICBDbGFzcyAgICAgICAgICAgICA6YTE0LCBhZnRlciBhMTMsIDEwZFxuICAgIEludGVyYWN0aW9uICAgICAgIDphMTUsIGFmdGVyIGExMywgMTBkXG4gICAgQWN0aXZpdHkgICAgICAgICAgOmExNiwgMjAyMC0xMS0zMCwgOGRcblxuc2VjdGlvbiBNQU5BR0VNRU5UXG4gICAgU3RhZmYgVXBza2lsbCAgOmFjdGl2ZSwgYTE3LCBhZnRlciBhMywgNDhkXG4gICAgTWFuYWdlbWVudCBDaGFydCAgOmRvbmUsIGExOCwgYWZ0ZXIgYTMsIDIwZCBcbiAgICBSaXNrIGFzc2Vzc21lbnQgICA6YWN0aXZlLCBhMTksIGFmdGVyIGE2LCAzMGRcblxuc2VjdGlvbiBJTVBMRU1FTlRBVElPTlxuICAgIENvcmUgQXJjaGl0ZWN0dXJlICA6YTMwLCBhZnRlciBhNCwgMndcbiAgICBSb3V0aW5nICYgTmF2aWdhdGlvbiAgOmEzMSwgYWZ0ZXIgYTMwLCAxd1xuICAgIENsb2NrICA6YTMyLCBhZnRlciBhMzEsIDNkXG4gICAgRGlzcGxheSAgOmEzMywgYWZ0ZXIgYTMyLCAxd1xuICAgIEJsdWV0b290aCAgOmEzNCwgYWZ0ZXIgYTMzLCAxd1xuICAgIFNldHRpbmdzIFNjcmVlbiAgOmEzNSwgYWZ0ZXIgYTM0LCAzZFxuICAgIFBvd2VyIExldmVscyAgOmEzNiwgYWZ0ZXIgYTM0LCAxd1xuICAgIEVDRyA6YTM3LCBhZnRlciBhMzQsIDF3XG4gICAgSGVhcnQgUmF0ZSA6YTM4LCBhZnRlciBhMzQsIDF3XG4gICAgTWFwcyA6YTM5LCBhZnRlciBhMzQsIDhkXG4gICAgUGVkb21ldGVyIDphNDAsIGFmdGVyIGEzNCwgMXdcbiAgICBTcE8yIDphNDEsIGFmdGVyIGEzNCwgMXdcblxuc2VjdGlvbiBJTlRFR1JBVElPTlxuICAgIFVzZXIgQWNjZXB0YW5jZSBUZXN0aW5nICA6YTQyLCBhZnRlciBhNDEsIDR3XG4gICAgU2VjdXJpdHkgVGVzdGluZyAgOmE0MywgYWZ0ZXIgYTQyLCAyd1xuICAgIERvY3VtZW50YXRpb24gIDphNDQsIGFmdGVyIGE0MSwgNXdcblxuc2VjdGlvbiBPUEVSQVRJT04gQU5EIE1BSU5URU5BTkNFXG4gICAgUHJvZHVjdCBMYXVuY2ggIDphMjIsIGFmdGVyIGE0MywgNWRcbiAgICBTdXBwb3J0ICA6YTE3LCBhZnRlciBhMjIsIDVkIiwibWVybWFpZCI6eyJ0aGVtZSI6ImRlZmF1bHQifSwidXBkYXRlRWRpdG9yIjpmYWxzZX0)

Much of the work after the core architecture, display, routing and Bluetooth can be parallelised. That's kind of the good part of this architecture. It's designed to allow for the app to be in small slices. We still hit Amdahl’s Law in the sync parts but it's still a reasonable speed up. We would still need to be careful staffing something like this. If we ended up with contention on any of the items we could end up with a decrease in development capacity, where developers are forced to wait on a file to be available, or merge the results after the fact (good old queueing theory and Kingman’s Formula... ).

```mermaid
gantt
title ECG Smartwatch Design
dateFormat YYYY-MM-DD

section ASSIGNMENT
    Group Setting :done, a1, 2020-10-16, 10d
    Topic Choice : done, a2, after a1 , 5d
    Project Overview : done, a3, 2020-10-30, 1d
    Report production :a4, after a12 , 16d
    Report release :a5, 2020-12-18 , 1d
    Design :a20, after a6, 40d

section SYSTEM
    System Architecture :active, a6, 2020-11-02 , 7d
    User Characteristics :active, a7, 2020-11-09 , 6d
    Assumptions :active, a8, 2020-11-09 , 3d

section REQUIREMENTS
    Functional :active, a9, 2020-11-09, 27d
    Non-Functional :active, a10, 2020-11-15, 21d

section DIAGRAMS
    Overall :active, a12, 2020-11-09, 25d
    Use Case :active, a13, 2020-11-13, 10d
    State :a20, after a13, 10d
    Class :a14, after a13, 10d
    Interaction :a15, after a13, 10d
    Activity :a16, 2020-11-30, 8d

section MANAGEMENT
    Staff Upskill :active, a17, after a3, 48d
    Management Chart :done, a18, after a3, 20d
    Risk assessment :active, a19, after a6, 30d

section IMPLEMENTATION
    Core Architecture  :a30, after a4, 2w
    Routing & Navigation  :a31, after a30, 1w
    Clock  :a32, after a31, 3d
    Display  :a33, after a32, 1w
    Bluetooth  :a34, after a33, 1w
    Settings Screen  :a35, after a34, 3d
    Power Levels  :a36, after a34, 1w
    ECG :a37, after a34, 1w
    Heart Rate :a38, after a34, 1w
    Maps :a39, after a34, 8d
    Pedometer :a40, after a34, 1w
    SpO2 :a41, after a34, 1w

section INTEGRATION @jdrew1303
    User Acceptance Testing  :a42, after a41, 4w
    Security Testing  :a43, after a42, 2w
    Documentation  :a44, after a41, 5w

section OPERATION AND MAINTENANCE
    Product Launch  :a22, after a43, 5d
    Support  :a17, after a22, 5d
```